hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -256,13 +258,13 @@ public void createVolume(String volumeName, VolumeArgs volArgs)
         OzoneQuota.parseQuota(volArgs.getQuota()).sizeInBytes();
     List<OzoneAcl> listOfAcls = new ArrayList<>();
     //User ACL
-    listOfAcls.add(new OzoneAcl(OzoneAcl.OzoneACLType.USER,
+    listOfAcls.add(new OzoneAcl(ACLIdentityType.USER,
```

Comment:
There is one small problem here. We (Ozone) is a superclass of POSIX ACLs. we have world and anonymous access defined differently. World users are still users we know -- that is we can log the user identity, anonymous is that we don't know and we don't care. I am not sure ACLIdentityType takes care of that. A similar comment was made earlier by @xiaoyuyao  since the ACLs itself is a larger group for us.

Reply:
We can address this by adding ANONYMOUS type in ACLIdentityType.